From 66dae24e44d6802ced542762b2324a4a8f3d41b8 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:45 -0400
Subject: [redhat] kabi: Adding symbol pv_irq_ops (arch/x86/kernel/paravirt.c)

Message-id: <97116526d52e4e8e75008be4514a97428d365bf2.1410510773.git.poros@redhat.com>
Patchwork-id: 95987
O-Subject: [PATCH RHEL7.1 165/233] kabi: Adding symbol pv_irq_ops (arch/x86/kernel/paravirt.c)
Bugzilla: 1062361 1071099
RH-Acked-by: Andrew Jones <drjones@redhat.com>

Bugzilla: 1062361,1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Refernced from spin_lock_irq.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/pv_irq_ops

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/pv_irq_ops b/redhat/kabi/kabi-module/kabi_x86_64/pv_irq_ops
new file mode 100644
index 0000000..72e458a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/pv_irq_ops
@@ -0,0 +1,2 @@
+#1-
+0x78764f4e pv_irq_ops vmlinux EXPORT_SYMBOL
-- 
1.7.1